modules/apps/calendar/calendar-web-test/src/testFunctional/tests/AddCalendarAccessibility.testcase

```diff
@@ -76,4 +76,35 @@ definition {
 		}
 	}
 
+	@description = "LPS-185089 Verifies that the focus is not lost after closing the 'Edit Event' modal."
+	@priority = 3
+	test FocusIsNotLostAfterClosingTheModal {
+		task ("And a new event is added") {
+			CalendarNavigator.gotoAddEvent();
+		}
+
+		task ("When the user tries to edit the event") {
+			CalendarEvent.editTitle(eventTitle = "Calendar Event Title");
+		}
+
+		task ("And clicks any 'Select' button on 'CATEGORIZATION'") {
+			CalendarEvent._expandPanel(panelName = "Categorization");
+
+			ScrollWebElementIntoView(locator1 = "Button#SELECT_CATEGORIES");
```

Comment:
I think that way is working fine.

Reply:
It's weird because I was using `ScrollWebElementIntoView` before and it was not working as expected, I was maybe applying it at another part of the code.

Reply:
Or maybe applying it before expanding `CATEGORIZATION` 🤔